### PR TITLE
Add autosignintokens to SAR

### PR DESCRIPTION
**Pull request by laurenemms**
With new autosignintokens table, this adds it to the SAR with tests.

### CODE REVIEW

File: app/models/database/postgres/PostgresSubjectAccessRequestRepository.scala [hunk 1]
```diff
@@ -122,4 +123,12 @@ class PostgresSubjectAccessRequestRepository @Inject()(val metricsActorProvider:
         .map(row => Json.prettyPrint(Json.toJson(row)))
     }(logFailure(s"failed to get newsletter subscriptions for $id"))
   }
+
+  def getAutoSignInTokens(id: String) = withMetricsFE("sar.getAutoSignInTokens") {
+      val sql =
+        sql"""
+             | select * from autosignintokens where identity_id = $id
```
Comment (jfsoul) [inline app/models/database/postgres/PostgresSubjectAccessRequestRepository.scala]: I think we should probably omit the token itself from the response, a bit like we have in some of the other queries (except we're not dealing with JSON this time).

File: test/repositories/postgres/PostgresSubjectAccessRequestRepositoryTest.scala [hunk 2]
```diff
@@ -122,5 +122,17 @@ class PostgresSubjectAccessRequestRepositoryTest extends WordSpecLike
...
+
+    "getAutoSignInTokens" in new TestFixture {
+      execSql(sql"INSERT INTO users values ('54321', '{}')")
+      execSql(
+        sql"""
+             | INSERT INTO autosignintokens VALUES ('123-token', '54321', '1234@email.com', '3000-01-01T00:00:00.000Z', false, false)
+         """.stripMargin)
+
+      whenReady(repo.getAutoSignInTokens("54321")) { result =>
+        result.map(_ shouldBe List("123-token"))
```
Comment (jfsoul) [inline test/repositories/postgres/PostgresSubjectAccessRequestRepositoryTest.scala]: does the result include all columns then or just the token itself?

File: app/models/database/postgres/PostgresSubjectAccessRequestRepository.scala (hunk 1 above)
Comment (laurenemms) [reply app/models/database/postgres/PostgresSubjectAccessRequestRepository.scala]: Okay, I'll leave the token out

File: test/repositories/postgres/PostgresSubjectAccessRequestRepositoryTest.scala (hunk 2 above)
Comment (laurenemms) [reply test/repositories/postgres/PostgresSubjectAccessRequestRepositoryTest.scala]: Have modified it and tests accordingly

**Pull request merged**